PortMaster/pylibs/harbourmaster/config.py

```diff
@@ -75,6 +90,15 @@
     HM_DEFAULT_PORTS_DIR = Path("/storage/roms/ports")
     HM_DEFAULT_SCRIPTS_DIR = Path("/storage/roms/ports")
 
+## Check if retrodeck.cfg exists. Chose this file/location as platform independent from were retrodeck is installed.
+elif (Path.home() / ".var/app/net.retrodeck.retrodeck/config/retrodeck/retrodeck.cfg").is_file():
+    retrodeck_roms_path = safe_cat(Path.home() / ".var/app/net.retrodeck.retrodeck/config/retrodeck/retrodeck.cfg")
```

Comment:
No need to include safe_cat, you can just do:
```python
 retrodeck_roms_path = (Path.home() / ".var/app/net.retrodeck.retrodeck/config/retrodeck/retrodeck.cfg").read_text()
 ```
You already verified the file exists. 😄